src/core/agent/records/operationPendingRecord.type.ts

```diff
@@ -2,4 +2,5 @@ export enum OperationPendingRecordType {
   Witness = "witness",
   Group = "group",
   Oobi = "oobi",
+  SubmitAdmitCredential = "submitAdmitCredential",
```

Comment:
Lets keep this as `Exchange = "exchange"` to match the oid of the operation. (operation ID)